includes/class-wc-calypso-bridge-setup.php

```diff
@@ -286,6 +286,41 @@ public function woocommerce_create_pages_callback() {
 
 				WC_Install::create_pages();
 
+				// Get navigation menu page and set up the menu.
+				$menu_page_post = get_page_by_path( 'primary', OBJECT, 'wp_navigation' );
+				if ( is_a( $menu_page_post, 'WP_Post' ) ) {
+
+					$menu_pages = array(
+						'shop'       => get_post( get_option( 'woocommerce_shop_page_id' ) ),
+						'blog'       => get_page_by_path( 'blog' ),
+						'my-account' => get_post( get_option( 'woocommerce_myaccount_page_id' ) ),
+						'contact-us' => get_page_by_path( 'contact-us' ),
+					);
+
+					$menu_content = '<!-- wp:navigation-link {"label":"' . __( 'Home', 'woocommerce' ) . '","url":"/","kind":"custom","isTopLevelLink":true} /-->';
+
+					foreach ( $menu_pages as $key => $page ) {
+						if ( ! is_a( $page, 'WP_Post' ) ) {
+							continue;
+						}
+
+						$title = $page->post_title;
```

Comment:
Is this safe for special chars? Like single and double quotes?

Reply:
Updated